scipy/stats/_stats_py.py

```diff
@@ -9335,6 +9335,29 @@ def friedmanchisquare(*samples):
     References
     ----------
     .. [1] https://en.wikipedia.org/wiki/Friedman_test
+    .. [2] P. Sprent and N.C. Smeeton, "Applied Nonparametric Statistical
+           Methods, Third Edition". Chapter 6, Section 6.3.2.
+
+    Examples
+    --------
+    For a group of seven students the pulse rate (per minute) was measured
+    before exercise, immediately after exercise and 5 minutes after exercise.
+    We use the Friedman test to determine whether the pulse rates are similar
+    on the three different occasions.
+
+    >>> from scipy.stats import friedmanchisquare
+    >>> before = [72, 96, 88, 92, 74, 76, 82]
+    >>> immediately_after = [120, 120, 132, 120, 101, 96, 112]
+    >>> five_min_after = [76, 95, 104, 96, 84, 72, 76]
+    >>> stat, p = friedmanchisquare(before, immediately_after, five_min_after)
```

Comment:
In between, it would be good to add some text saying what is the null hypothesis, etc. Again have a look at other examples like `levene` (the plotting part is not necessary here.)

`chi2_contingency` is also a good example.

Looking at these, you should try to follow the same wording. To describe the results, etc.